<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>customerpage</class>
 <widget class="QDialog" name="customerpage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>610</width>
    <height>531</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QPushButton" name="addButton">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>230</y>
     <width>281</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>Add to cart</string>
   </property>
  </widget>
  <widget class="QTabWidget" name="tabWidge">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>10</y>
     <width>261</width>
     <height>211</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>3</number>
   </property>
   <property name="usesScrollButtons">
    <bool>true</bool>
   </property>
   <property name="documentMode">
    <bool>true</bool>
   </property>
   <property name="tabsClosable">
    <bool>false</bool>
   </property>
   <widget class="QWidget" name="tab">
    <attribute name="title">
     <string>Pharm</string>
    </attribute>
    <attribute name="toolTip">
     <string>Pharm section</string>
    </attribute>
    <widget class="QTableView" name="tableProducts">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>261</width>
       <height>181</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_2">
    <attribute name="title">
     <string>Food</string>
    </attribute>
    <attribute name="toolTip">
     <string>Dairy section</string>
    </attribute>
    <widget class="QTableView" name="tableDairy">
     <property name="geometry">
      <rect>
       <x>-5</x>
       <y>0</y>
       <width>381</width>
       <height>181</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_5">
    <attribute name="title">
     <string>Cleaning</string>
    </attribute>
    <attribute name="toolTip">
     <string>Cleaning section</string>
    </attribute>
    <widget class="QTableView" name="tableCleaning">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>371</width>
       <height>181</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_6">
    <attribute name="title">
     <string>Alcohol</string>
    </attribute>
    <attribute name="toolTip">
     <string>Alcohol section</string>
    </attribute>
    <widget class="QTableView" name="tableAlcohol">
     <property name="geometry">
      <rect>
       <x>-5</x>
       <y>0</y>
       <width>391</width>
       <height>181</height>
      </rect>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QTableView" name="table_cart">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>290</y>
     <width>271</width>
     <height>201</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="delete_button">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>320</y>
     <width>113</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>Delete Item</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>500</y>
     <width>81</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Total Price:</string>
   </property>
  </widget>
  <widget class="QLabel" name="TotalPriceValue">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>500</y>
     <width>81</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>20</y>
     <width>60</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Pruducts:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>320</y>
     <width>60</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>My cart:</string>
   </property>
  </widget>
  <widget class="QPushButton" name="checkout">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>390</y>
     <width>113</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>Checkout</string>
   </property>
  </widget>
 </widget>
 <tabstops>
  <tabstop>tabWidge</tabstop>
  <tabstop>addButton</tabstop>
  <tabstop>tableProducts</tabstop>
  <tabstop>tableDairy</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
